handlers/main.yml

```diff
@@ -6,3 +6,7 @@
 - name: restart syncthing
   command: systemctl restart syncthing
   when: syncthing_use_systemd
+
+- name: restart syncthing
+  command: service restart syncthing
```

Comment:
This should not be necessary. Looking at the documentation of ansible [`service`](http://docs.ansible.com/ansible/service_module.html) module, it automatically chooses the available init systems. Something like this should now be sufficient:
```
- name: restart syncthing
  service: restart syncthing
```